test/uploads/e2e.spec.ts

```diff
@@ -288,18 +286,17 @@ describe('uploads', () => {
         await expect(page.locator('.edit-upload__input input[name="X %"]')).toHaveValue(`${focalX}`)
         await expect(page.locator('.edit-upload__input input[name="Y %"]')).toHaveValue(`${focalY}`)
 
+        // apply crop
         await page.locator('button:has-text("Apply Changes")').click()
         await page.waitForSelector('button#action-save')
         await page.locator('button#action-save').click()
+        await wait(1000) // Wait for the save
```

Comment:
Is there a toast for this operation that we can listen in on? If so, we can use something like 

```
await expect(page.locator('.Toastify__toast--success')).toHaveText(
  'Deleted 3 Posts successfully.',
)
```

Reply:
This worked - adding this check.
```javascript
await expect(page.locator('.Toastify')).toContainText('successfully')
```